### ISSUE TITLE
pytest is broken

### ISSUE THREAD

**Issue by jugmac00** [FlaskCon/traveller#29]
**reason**

There is a package called `y` with tests files with invalid Python syntax. cc @Abdur-rahmaanJ 

```
❯ pytest
========================================================== test session starts ===========================================================
platform linux -- Python 3.8.11, pytest-6.1.1, py-1.10.0, pluggy-0.13.1
rootdir: /home/jugmac00/Projects/traveller/traveller
plugins: cov-2.11.1, Faker-7.0.1, order-0.9.2, dotenv-0.5.2
collected 104 items / 10 errors / 94 selected                                                                                            

================================================================= ERRORS =================================================================
_______________________________________ ERROR collecting modules/cfp/tests/test_cfp_functional.py ________________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/cfp/tests/test_cfp_functional.py", line 2
E       Please add your functional tests to this file.
E              ^
E   SyntaxError: invalid syntax
_________________________________________ ERROR collecting modules/cfp/tests/test_cfp_models.py __________________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/cfp/tests/test_cfp_models.py", line 2
E       Please add your models tests to this file.
E              ^
E   SyntaxError: invalid syntax
______________________________________ ERROR collecting modules/conf/tests/test_conf_functional.py _______________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/conf/tests/test_conf_functional.py", line 2
E       Please add your functional tests to this file.
E              ^
E   SyntaxError: invalid syntax
________________________________________ ERROR collecting modules/conf/tests/test_conf_models.py _________________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/conf/tests/test_conf_models.py", line 2
E       Please add your models tests to this file.
E              ^
E   SyntaxError: invalid syntax
___________________________________ ERROR collecting modules/profile/tests/test_profile_functional.py ____________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/profile/tests/test_profile_functional.py", line 2
E       Please add your functional tests to this file.
E              ^
E   SyntaxError: invalid syntax
_____________________________________ ERROR collecting modules/profile/tests/test_profile_models.py ______________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/profile/tests/test_profile_models.py", line 2
E       Please add your models tests to this file.
E              ^
E   SyntaxError: invalid syntax
__________________________________ ERROR collecting modules/schedule/tests/test_schedule_functional.py ___________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/schedule/tests/test_schedule_functional.py", line 2
E       Please add your functional tests to this file.
E              ^
E   SyntaxError: invalid syntax
____________________________________ ERROR collecting modules/schedule/tests/test_schedule_models.py _____________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/schedule/tests/test_schedule_models.py", line 2
E       Please add your models tests to this file.
E              ^
E   SyntaxError: invalid syntax
_________________________________________ ERROR collecting modules/y/tests/test_y_functional.py __________________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/y/tests/test_y_functional.py", line 2
E       Please add your functional tests to this file.
E              ^
E   SyntaxError: invalid syntax
___________________________________________ ERROR collecting modules/y/tests/test_y_models.py ____________________________________________
../venv/lib/python3.8/site-packages/_pytest/python.py:571: in _importtestmodule
    mod = import_path(self.fspath, mode=importmode)
../venv/lib/python3.8/site-packages/_pytest/pathlib.py:517: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../venv/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:357: in _rewrite_test
    tree = ast.parse(source, filename=fn_)
/usr/lib/python3.8/ast.py:47: in parse
    return compile(source, filename, mode, flags,
E     File "/home/jugmac00/Projects/traveller/traveller/modules/y/tests/test_y_models.py", line 2
E       Please add your models tests to this file.
E              ^
E   SyntaxError: invalid syntax
======================================================== short test summary info =========================================================
ERROR modules/cfp/tests/test_cfp_functional.py
ERROR modules/cfp/tests/test_cfp_models.py
ERROR modules/conf/tests/test_conf_functional.py
ERROR modules/conf/tests/test_conf_models.py
ERROR modules/profile/tests/test_profile_functional.py
ERROR modules/profile/tests/test_profile_models.py
ERROR modules/schedule/tests/test_schedule_functional.py
ERROR modules/schedule/tests/test_schedule_models.py
ERROR modules/y/tests/test_y_functional.py
ERROR modules/y/tests/test_y_models.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 10 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=========================================================== 10 errors in 1.14s ===========================================================

```

**Comment by jugmac00** [comment]
I think this will get fixed with #26

**Comment by jugmac00** [comment]
fixed by #26
